app/main/views/questionnaire.py

```diff
@@ -119,13 +125,31 @@ def submit_answers(eq_id, form_type, collection_id):
         return redirect_to_questionnaire_page(eq_id, form_type, collection_id, invalid_location)
 
 
+@action_blueprint.route('<location>/<question>/add', methods=["POST"])
+@login_required
+def add_answer(eq_id, form_type, collection_id, location, question):
+    answer_store = get_answer_store(current_user)
```

Comment:
we are just loading answer store from global to pass it into questionnaire_manager. should we just load it in questionnaire_manager?